spec/lexers/puppet_spec.rb

```diff
@@ -3,6 +3,55 @@
 describe Rouge::Lexers::Puppet do
   let(:subject) { Rouge::Lexers::Puppet.new }
 
+  describe 'lexing' do
```

Comment:
Thanks for writing some tests. This isn't necessary, though. Simply adding an example in `samples/puppet` is sufficient.